src/frontend/next/src/components/SnackBar/SnackBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC, useState } from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+
+type SnackbarProps = {
+  message: string;
+};
+
+const SimpleSnackbar: FC<SnackbarProps> = (props) => {
+  const { message } = props;
+  const [open, setOpen] = useState<boolean>(true);
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <div>
```

Comment:
`<div>` vs `<>`, are we going to give this a `className` sometime?

Reply:
We might not but, I tried to keep everything as original as possible. We might not need that <div>

Reply:
I think we should keep the structure of the component the same.